src/documentation_server.ml

```diff
@@ -16,6 +16,7 @@ let debouncing_middleware ~timeout_in_ms f =
 let start ~path ?(port = 0) () =
   Promise.make @@ fun ~resolve ~reject:_ ->
   let polka = Polka.create () in
+  (* Closes the server after some idle time *)
```

Comment:
What's the need to do this though? The fact that the server is turned off idling isn't at all obvious, and the resources saved by this don't see very impressive either. If it's so important to turn this off, let's just allow the user to do this manually with a status item element.